http-clients/netty-nio-client/src/main/java/software/amazon/awssdk/http/nio/netty/internal/nrs/HttpStreamsHandler.java

```diff
@@ -211,7 +213,7 @@ private void handleCancelled(ChannelHandlerContext ctx, InT msg) {
     private void handleReadHttpContent(ChannelHandlerContext ctx, HttpContent content) {
         if (!ignoreBodyRead) {
             if (content instanceof LastHttpContent) {
-
+                ctx.channel().attr(LAST_HTTP_CONTENT_RECEIVED_KEY).set(true);
```

Comment:
Do we want to move the logging statement over too?

Reply:
done, added Debug log same as before